### ISSUE TITLE
What is the role of black holes in the Universe? (proposed key question)

### ISSUE THREAD

**Issue by ethrane** [ethrane/transients#2]
@ethrane and @kauchettl propose this as a key question. The question would cover various topics like tidal disruption events, gravitational waves from merging black holes, AGN flares, etc. The question is much broader than the domain of our transient group. In particular, there are strong connections to Group 1.1 (Galaxies & Cosmology) who are interested in black hole feedback, gravitational waves from supermassive black holes, etc. There was no question in last decadal plan specifically about black holes, but ten years later, black holes are having a moment. This question seems timely.

Update on 28 Feb: I implemented Clancy's suggestion: "the role of black holes in the e̶v̶o̶l̶u̶t̶i̶o̶n̶ ̶o̶f̶ ̶t̶h̶e̶ ̶ Universe."

**Comment by cwjames1983** [comment]
I'd like to add that neutrino and cosmic ray work could be included in this one, since AGN (and long GRB!) are strong candidate sources. I'd suggest a title without "evolution of the Universe" in it though, since that sounds a bit too cosmological.

**Comment by ethrane** [comment]
Good call, @cwjames1983: the original wording was too cosmological. I've implemented your suggestion.

**Comment by Ch-Wolf** [comment]
❤️
